app/assets/javascripts/application.js

```diff
@@ -135,8 +135,9 @@ $(document).on('ready', function() {
     });
   });
 
-
-  $('.js-first-visit-notice').on('close.bs.alert', async () => {
-    document.cookie = 'dismiss_fvn=true; path=/; expires=Fri, 31 Dec 9999 23:59:59 GMT';
-  });
+  if (document.cookie.indexOf ('dismiss_fvn') == -1 ) {
```

Comment:
**suggestion** (code-issues, blocking): `===` for equality checks in JS other than `== undefined`

**nitpick** (code-style, non-blocking): no space before opening paren in function call, i.e. `indexOf('dismiss_fvn')`

Reply:
All changes in this review comment were implemented